tests/base.py

```diff
@@ -222,14 +223,27 @@ def modify_expected_datatypes(self, expected_records):
             for record in expected_records:
                 for key, value in record.items():
                     self.align_date_type(record, key, value)
+                    self.sort_array_type(record, key, value)
             return
 
         for key, value in expected_records.items(): # Modify a single record
             self.align_date_type(expected_records, key, value)
+            self.sort_array_type(expected_records, key, value)
 
     def align_date_type(self, record, key, value):
         """datetime values must conform to ISO-8601 or they will be rejected by the gate"""
         if isinstance(value, str) and key in ['updated_at']:
             raw_date = self.parse_date(value)
             iso_date = dt.strftime(raw_date,  "%Y-%m-%dT%H:%M:%S.%fZ")
             record[key] = iso_date
+
+    def sort_array_type(self, record, key, value):
```

Comment:
Yes we are sorting arrays in order to make a valid comparison between expected and actual records. It is not always necessary but in this case square returns fields that are arrays as unordered lists. Since we are passing in the whole record set, we are checking the type.

Reply:
Ok, that makes sense. Can you write a docstring explaining that?

Reply:
Done